rareapi/views/categories.py

```diff
@@ -1,14 +1,53 @@
 """View module for handling requests about game types"""
+from django.core.exceptions import ValidationError
+from rest_framework import status
 from django.http import HttpResponseServerError
 from rest_framework.viewsets import ViewSet
 from rest_framework.response import Response
 from rest_framework import serializers
-from rareapi.models import Categories
+from rareapi.models import Categories, categories
 
 
 class CategoriesViewset(ViewSet):
     """rare restful Categories"""
 
+    def create(self, request):
+        """Handle POST operations
+
+        Returns:
+            Response -- JSON serialized category instance
+        """
+
+        # Uses the token passed in the `Authorization` header
+        categories = Categories.objects.get(user=request.auth.user)
```

Comment:
Since Categories aren't related to users lines 21-22 are breaking creating a category